sys/i2c/i2c_io.c

```diff
@@ -63,7 +72,20 @@ error_code_t i2cReceiveFrom(uint8_t sAddr, uint8_t *buf, uint16_t numBytes) {
   }
 
   /* Mock the receive using getLm75bdNextTempRegVal() */
-
+  if(sAddr == LM75BD_I2C_ADDR){
+    uint16_t recv_buff = getLm75bdNextTempRegVal();
+    memset(buf, 0, numBytes);
+    if(numBytes <= sizeof(recv_buff)){
+      memcpy(buf, &recv_buff, numBytes);
+    } 
+    else {
+      memcpy(buf, &recv_buff, sizeof(recv_buff));
+      // exit or throw error instead?
+    }
```

Comment:
The received data depends on the reg address that was sent in the last `i2cSendTo`. So maybe do a check to make sure it was the temp reg?